README.md

```diff
@@ -18,32 +18,57 @@ In order to run the tests in this repository, please clone each of the following
 5. Clone the `main` branch of https://github.com/buerokratt/Tests to  `[working folder]/Tests`
 
-### Install .net CLI
+### 2 Install .net CLI
 
 The tests are written using .net so you need the .net CLI to run them.
 
 1. Install the .net 6.0 (or newer) CLI from [Install .NET on Windows, Linux, and macOS](https://docs.microsoft.com/en-us/dotnet/core/install/)
 
-### Install Docker
+### 3 Install Docker
 
 When running tests locally, you will use Docker Desktop.
 
 1. Install Docker Desktop from https://www.docker.com/products/docker-desktop/
 
-### Docker Network Bridge
+### 4 Docker Network Bridge
 
 In order to run the various services in containers concurrently, you will use Docker Compose. In order for the containers to be able to communicate with each other, you will create a Docker network bridge.
 
 1. Open a terminal and navigating to `[working folder]`
 2. Execute `docker network create -d bridge my-network`
 
-### Docker Compose
+### 5 Docker Compose
 
 To run the containers, you will need a Docker Compose file.
 
 1. Open a terminal and navigate cd to `[working folder]/Tests`
 3. Execute `docker compose up --build` to build and run the containers specified in the `docker-compose.yml` file. This has completed when you see "created" next to each of the 4 containers and you see logs in the terminal
 
+### 6 Run Test
+
+You can run the test in a terminal or Visual Studio. These steps are for a terminal but you can get a more detailed view by running the tests in test Explorer in Visual Studio.
+
+> The test will use the version of the code which is on your local device for each repository/container. Generally, you'll want to make sure you have pulled the latest changes on the `main` branch before running the test, although there may be scenarios where you want to test from other branches.
+
+1. Open a terminal and navigate to `[working folder]/Tests/src/`
+2. Execute `dotnet test`
+3. If successful, you will see that 2 tests from "Tests.IntegrationTests.dll" has "Passed!" in the terminal.
+
+## Running tests locally against Azure hosted environment
+
+You may want to run this test against deployed component in Azure. This can be usefull to make sure that the deployed components have the correct settings and can communicate correctly.
+
+In order to do this follow steps 1 and 2 from Running tests locally with Docker Compose and then follow these steps:
+
+### Configure secrets.json
+
+In order to target a deployed set of resources, the settings in `appsettings.json` need to be over-written with values that corespond to the deployed components. It is not good practice to edit `appsettings.json` because that file is under source control and you may inadvertently commit secrets to Git. Instead you can use .net user secrets to create a local secrets.json which over-rides the settings in `appsettings.json`. See https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-6.0
```

Comment:
```suggestion
In order to target a deployed set of resources, the settings in `appsettings.json` need to be over-written with values that correspond to the deployed components. It is not good practice to edit `appsettings.json` because that file is under source control and you may inadvertently commit secrets to Git. Instead you can use .net user secrets to create a local secrets.json which over-rides the settings in `appsettings.json`. See https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-6.0
```

Reply:
How you can spot these and the Typora spell check cannot is beyond me. Done, thanks! :)